frontend/simple/views/GroupDashboard.vue

```diff
@@ -72,6 +74,7 @@ import GroupMembers from './containers/GroupMembers.vue'
 import SupportHistory from './components/Graphs/SupportHistory.vue'
 import GroupSettings from './components/GroupSettings.vue'
 import ProgressOverview from './components/ProgressOverview.vue'
+import ProposalsForm from './containers/proposals-form/'
```

Comment:
In my PR I've gone ahead and, per the reasoning of one of the [10 Regrets I have about Node.js](https://www.youtube.com/watch?v=M3BM9TB-8yA) video, made explicit the extension of all includes, so please go through this PR and check to make sure that all files that are imported have their extensions fully specified

Reply:
I saw your PR before reading this comment and totally agreed with explicit extensions. I never liked to have a "ghost" path and couldn't fully argue the whys of that. With that talk I now can :D
I'll change it on this PR.